packaging/installer-linux/installer-debian/src/main/resources/common/neo4j

```diff
@@ -46,6 +47,14 @@ SCRIPTNAME=/etc/init.d/$NAME-service
 
 do_start()
 {
+  # Make sure that /var/run/neo4j exists
+  [ -d "${PIDDIR}" ] || mkdir -p "${PIDDIR}"
+
+  # And that it has correct permissions
+  if [ "${NEO_USER}" != "root" ]; then
+    chown "${NEO_USER}" "${PIDDIR}"
```

Comment:
I habitually set the group as well as user in cases like this (`chown "${NEO_USER}:"`).